### PR TITLE
Reference management models content v9

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Belongs to #3145

### CODE REVIEW

**Comment by busrasengul** [comment]
Thank you @erikjanwestendorp
It looks great as always! We'll get this review on our next meeting!
#h5yr

File: Reference/Management/Models/Content-v9.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
+---
+versionFrom: 9.0.0
+verified-against: rc001
+meta-title: Content Model
```
Comment (busrasengul) [inline Reference/Management/Models/Content-v9.md:4]:
```suggestion
meta.Title: "Content Model"
```

File: Reference/Management/Models/Content-v9.md [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+meta.Description: The Content class represents a single item in the content tree, its values are fetched directly from the database, not from the cache.
```
Comment (busrasengul) [inline Reference/Management/Models/Content-v9.md:5]:
```suggestion
meta.Description: "The Content class represents a single item in the content tree, its values are fetched directly from the database, not from the cache." 
```

File: Reference/Management/Models/Content-v9.md [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+
+Constructor for creating a new Content object where the necessary parameters are the name of the Content, the parent of the Content as an `IContent` object and the ContentType as an `IContentType` object for the Content being created. In addition, there is an optional parameter for the culture.
+
+### new Content(string name, IContent parent, IContentType contentType, int userId, string culture = null)
+
+Constructor for creating a new Content object where the necessary parameters are the name of the Content, the parent of the Content as an `IContent` object, the ContentType as an `IContentType` object for the Content being created and the id of the user as a `int`. In addition, there is an optional parameter for the culture.
+
+### new Content(string name, IContent parent, IContentType contentType, PropertyCollection properties, string culture = null)
+
+Constructor for creating a new Content object where the necessary parameters are the name of the Content, the parent of the Content as an `IContent` object, the ContentType as an `IContentType` object and a `PropertyCollection` for the Content being created.In addition, there is an optional parameter for the culture.
```
Comment (busrasengul) [inline Reference/Management/Models/Content-v9.md:40]:
```suggestion
Constructor for creating a new Content object where the necessary parameters are the name of the Content, the parent of the Content as an `IContent` object, the ContentType as an `IContentType` object and a `PropertyCollection` for the Content being created. In addition, there is an optional parameter for the culture.
```

**Review by busrasengul — CHANGES_REQUESTED** [review]

File: Reference/Management/Models/Content-v9.md (hunk 3 above)
Comment (erikjanwestendorp) [reply Reference/Management/Models/Content-v9.md]: @busrasengul Updated

**Pull request merged**